src/Nethermind/Nethermind.Core.Test/Blockchain/TestBlockchain.cs

```diff
@@ -377,7 +377,8 @@ protected virtual async Task AddBlocksOnStart()
             State,
             ReceiptStorage,
             new BlockhashStore(SpecProvider, State),
-            LogManager);
+            LogManager,
+            preWarmer: new BlockCachePreWarmer(new ReadOnlyTxProcessingEnvFactory(WorldStateManager, BlockTree, SpecProvider, LogManager, WorldStateManager.GlobalWorldState), SpecProvider, LogManager, WorldStateManager.GlobalWorldState));
```

Comment:
Add method to create pre-warmer in TestBlockchain class, so we don't have to duplicate this code and just call that method?